### ISSUE TITLE
MiniMessage support

### ISSUE THREAD

**Issue by SeemWind** [knokko/custom-items-gradle#307]
https://docs.advntr.dev/minimessage/index.html

make color code and gradient convenient

**Comment by knokko** [comment]
Where would you like to use mini messages? Item display names?

**Comment by SeemWind** [comment]
Yes, and lores

**Comment by SeemWind** [comment]
I have mentioned the issue with gradient code before. Perhaps you have encountered some difficulties, but minimassage should be simpler

**Comment by knokko** [comment]
I don't think you have ever mentioned gradient code to me. At least, I don't remember it, and searching this repository and the discord server for "gradient" yields 0 results.

**Comment by SeemWind** [comment]
My fault, I just have said hex color
![image](https://github.com/knokko/custom-items-gradle/assets/50366675/055e02f7-1673-4d33-82aa-40883b0f06d0)

**Comment by SeemWind** [comment]
But still a way to resolve

**Comment by knokko** [comment]
I haven't really looked into hex colors yet. There are simply more Github issues being created than I can implement, so some issues are not picked up for a long time.
Should I skip hex colors if I decide to implement this instead?

**Comment by SeemWind** [comment]
This should be possible, as it provides complete functionality for text components. If you want to be compatible with old format code, such as &, it also provides methods

**Comment by SeemWind** [comment]
Bold and italics can also be implemented

**Comment by SeemWind** [comment]
By the way, the text edit box should have the ability to move the cursor, at least can copy the text, please, which make edit easier

**Comment by knokko** [comment]
You can already copy the text with Control + C or Control + X. Actually moving the cursor will have to wait for an editor rewrite
